lms/static/sass/discussion/utilities/_v1-compatibility.scss

```diff
@@ -0,0 +1,198 @@
+// Utilities to provide v1-styling compatibility
+
+@mixin font-size($sizeValue: 16){
+  font-size: $sizeValue + px;
+}
+
+@mixin line-height($fontSize: auto){
+  line-height: ($fontSize*1.48) + px;
+}
+
+%text-sr {
+    // clip has been deprecated but is still supported
+    clip: rect(1px 1px 1px 1px);
+    clip: rect(1px, 1px, 1px, 1px);
+    position: absolute;
+    margin: -1px;
+    height: 1px;
+    width: 1px;
+    border: 0;
+    padding: 0;
+    overflow: hidden;
+    // ensure there are spaces in sr text
+    word-wrap: normal;
+}
+
+// extends - UI - depth levels
+%ui-depth0 { z-index: 0; }
+%ui-depth1 { z-index: 10; }
+%ui-depth2 { z-index: 100; }
+%ui-depth3 { z-index: 1000; }
+%ui-depth4 { z-index: 10000; }
+%ui-depth5 { z-index: 100000; }
+
+%t-icon1 {
+  @include font-size(48);
+}
+
+%t-icon2 {
+  @include font-size(36);
+}
+
+%t-icon3 {
+  @include font-size(24);
+}
+
+%t-icon4 {
+  @include font-size(18);
+}
+
+%t-icon5 {
+  @include font-size(16);
+}
+
+%t-icon6 {
+  @include font-size(14);
+}
+
+%t-icon7 {
+  @include font-size(12);
+}
+
+%t-icon8 {
+  @include font-size(11);
+}
+
+%t-icon9 {
```

Comment:
(Room for debate on this, it reduces readability/parseability to some extent but makes the file cleaner, IMO)